libmachine/provision/arch.go

```diff
@@ -154,7 +154,7 @@ func (provisioner *ArchProvisioner) Provision(swarmOptions swarm.SwarmOptions, a
 	}
 
 	log.Debug("Configuring swarm")
-	if err := configureSwarm(provisioner, swarmOptions, provisioner.AuthOptions); err != nil {
+	if err := configureSwarm(provisioner, provisioner.EngineOptions.Env, swarmOptions, provisioner.AuthOptions); err != nil {
```

Comment:
Instead of adding an additional argument to this function, please just add a `Env` value to the `SwarmOptions` struct and it will get passed through automatically.